client/express-checkout/index.js

```diff
@@ -526,48 +521,6 @@ jQuery( ( $ ) => {
 			wcpayECE.init();
 		},
 
-		blockExpressCheckoutButton: () => {
-			// check if element isn't already blocked before calling block() to avoid blinking overlay issues
-			// blockUI.isBlocked is either undefined or 0 when element is not blocked
-			if (
-				$( '#wcpay-express-checkout-element' ).data(
-					'blockUI.isBlocked'
-				)
-			) {
-				return;
-			}
-
-			$( '#wcpay-express-checkout-element' ).block( { message: null } );
-		},
-
-		unblockExpressCheckoutButton: () => {
```

Comment:
Moved basically without changes to the `expressCheckoutButtonUi.unblockButton()` method.